src/neo4j_genai/indexes.py

```diff
@@ -238,10 +238,11 @@ def drop_index_if_exists(
 
 def upsert_vector(
```

Comment:
I think we can also add tests for upsert_vector for the relationship case

Reply:
Good point, I'll add tests.

Reply:
Done!